source/exec_cmd.c

```diff
@@ -94,14 +105,13 @@ static void zipSave(const save_entry_t* entry, const char* exp_path)
 
 	init_loading_screen("Exporting save game...");
 
-	fid = get_filename_id(exp_path, entry->title_id);
-	snprintf(export_file, sizeof(export_file), "%s%s-%08d.zip", exp_path, entry->title_id, fid);
+	snprintf(zip_file, sizeof(zip_file), "%s%s_%d-%02d-%02d_%02d%02d%02d.zip", exp_path, entry->dir_name, t.tm_year+1900, t.tm_mon+1, t.tm_mday, t.tm_hour, t.tm_min, t.tm_sec);
```

Comment:
I think for filename it might be better to do something like `TITLEID-Folder-etc`, so when you list files, it would be grouped by title id. 🤔 
Something like:

```suggestion
	snprintf(zip_file, sizeof(zip_file), "%s%s-%s_%d-%02d-%02d_%02d%02d%02d.zip", exp_path, entry->title_id, entry->dir_name, t.tm_year+1900, t.tm_mon+1, t.tm_mday, t.tm_hour, t.tm_min, t.tm_sec);
```

Reply:
Added